code/tabs.py

```diff
@@ -11,5 +11,10 @@ def tab_final():
         """Jumps to the final tab"""
 
     def tab_close_wrapper():
-        """Closes the current tab"""
+        """
+        Closes the current tab.
```

Comment:
Unrelated comment here that I forgot to add to a previous PR. Happy to put in a fresh PR if you prefer.